dev/pyproject.py

```diff
@@ -0,0 +1,161 @@
+                    # mlserver. Reference issue: https://github.com/SeldonIO/MLServer/issues/1089
+                    "mlserver>=1.2.0,!=1.3.1",
+                    "mlserver-mlflow>=1.2.0,!=1.3.1",
+                    "virtualenv",
+                    # Required for exporting metrics from the MLflow server to Prometheus
+                    # as part of the MLflow server monitoring add-on
+                    "prometheus-flask-exporter",
+                ],
+                "databricks": [
+                    # Required to write model artifacts to unity catalog locations
+                    "azure-storage-file-datalake>12",
+                    "google-cloud-storage>=1.30.0",
+                    "boto3>1",
+                    "botocore",
+                ],
+                "gateway": gateways_requirements,
+                "genai": gateways_requirements,
+                "sqlserver": ["mlflow-dbstore"],
+                "aliyun-oss": ["aliyunstoreplugin"],
+                "xethub": ["mlflow-xethub"],
+            },
+            "urls": {
+                "homepage": "https://mlflow.org",
+                "issues": "https://github.com/mlflow/mlflow/issues",
+                "documentation": "https://mlflow.org/docs/latest/index.html",
+                "repository": "https://github.com/mlflow/mlflow",
+            },
+            "scripts": {
+                "mlflow": "mlflow.cli:cli",
+            },
+            "entry-points": {
+                "mlflow.app": {
+                    "basic-auth": "mlflow.server.auth:create_app",
+                },
+                "mlflow.app.client": {
+                    "basic-auth": "mlflow.server.auth.client:AuthServiceClient",
+                },
+                "mlflow.deployments": {
+                    "databricks": "mlflow.deployments.databricks",
+                    "http": "mlflow.deployments.mlflow",
+                    "https": "mlflow.deployments.mlflow",
+                    "openai": "mlflow.deployments.openai",
+                },
+            },
+        },
+        "tool": {
+            "setuptools": {
+                "packages": {
+                    "find": {
+                        "where": ["."],
+                        "include": ["mlflow", "mlflow.*"],
+                        "exclude": ["tests", "tests.*"],
+                    }
+                },
+                "package-data": {
+                    "mlflow": [
+                        "store/db_migrations/alembic.ini",
+                        "temporary_db_migrations_for_pre_1_users/alembic.ini",
+                        "pypi_package_index.json",
+                        "pyspark/ml/log_model_allowlist.txt",
+                        "server/auth/basic_auth.ini",
+                        "server/auth/db/migrations/alembic.ini",
+                        "recipes/resources/*",
+                        "recipes/cards/templates/*",
```

Comment:
Miss `mlflow/` prefix? Or does it work as well?

Reply:
`mlflow/` must be stripped.

Reply:
stripped!